IPs/RSKIP428.md

```diff
@@ -0,0 +1,97 @@
+---
+rskip: 428
+title: HSM Segwit Support
+created: 23-APR-24
+author: NC
+purpose: Sca
+layer: Core 
+complexity: 2
+status: Draft
+description: 
+---
+
+|RSKIP          | 428                |
+| :------------ |:-------------------|
+|**Title**      | Support HSM Segwit |
+|**Created**    | 23-APR-24          |
+|**Author**     | NC                 |
+|**Purpose**    | Sca                |
+|**Layer**      | Core               |
+|**Complexity** | 2                  |
+|**Status**     | Draft              |
+
+## Abstract
+
+This RSKIP proposes creating a new event that stores the outpoint values of the inputs used in a 
+peg-out to enable HSM Segwit support in RSK.
```

Comment:
`HSM Segwit support` sounds a bit ambiguous. I'd try to be more clear. Explain that the goal is to provide the utxos value so that the PowHSM can build the sighash of a segwit transaction in order to sign it